src/assets/locales/fr.json

```diff
@@ -347,36 +384,43 @@
-  "preferences-screen.preference.animate-waypoint-transitions": "Animer les transitions entre les waypoint",
-  "preferences-screen.preference.disable-auto-gain-control": "Désactiver le contrôle automatique du gain du micro",
+  "preferences-screen.preference.animate-waypoint-transitions": "Animer les transitions entre les waypoints",
+  "preferences-screen.preference.audio-clipping-threshold": "Seuil pour l'audio clipping",
+  "preferences-screen.preference.disable-auto-gain-control": "Désactiver le contrôle automatique du volume du micro",
   "preferences-screen.preference.disable-auto-pixel-ratio": "Désactiver les ajustements automatiques du ratio de pixels",
   "preferences-screen.preference.disable-backwards-movement": "Désactiver les mouvements en marche arrière",
   "preferences-screen.preference.disable-echo-cancellation": "Désactiver l'annulation d'écho du micro",
   "preferences-screen.preference.disable-idle-detection": "Désactiver la sortie automatique si inactif ou en arrière-plan",
   "preferences-screen.preference.disable-movement": "Désactiver le mouvement",
   "preferences-screen.preference.disable-noise-suppression": "Désactiver la suppression de bruit du micro",
+  "preferences-screen.preference.disable-panning": "Désactiver le panning audio gauche/droite",
   "preferences-screen.preference.disable-sound-effects": "Désactiver les effets sonores",
   "preferences-screen.preference.disable-strafing": "Désactiver le mitraillage",
   "preferences-screen.preference.disable-teleporter": "Désactiver la téléportation",
+  "preferences-screen.preference.enable-audio-clipping": "Activer l'audio clipping",
   "preferences-screen.preference.enable-dynamic-shadows": "Activer les ombres dynamiques",
   "preferences-screen.preference.enable-gyro": "Activer le gyroscope (si compatible avec le navigateur / l'appareil)",
   "preferences-screen.preference.enable-on-screen-joystick-left": "Activer le joystick sur l'écran à gauche pour se déplacer",
   "preferences-screen.preference.enable-on-screen-joystick-right": "Activer le joystick sur l'écran à droite pour se déplacer",
+  "preferences-screen.preference.fast-room-switching": "Activer le basculement rapide de salle",
   "preferences-screen.preference.global-media-volume": "Volume audio des médias",
   "preferences-screen.preference.global-voice-volume": "Volume audio de la voix entrante",
   "preferences-screen.preference.invert-touchscreen-camera-move": "Inverser la direction du mouvement de caméra pour les écrans tactiles",
+  "preferences-screen.preference.lazy-load-scene-media": "Activer le chargement asynchrone des ressources multimédia de la scène",
   "preferences-screen.preference.locale": "Langue",
-  "preferences-screen.preference.material-quality-setting": "Qualité du matériau",
+  "preferences-screen.preference.material-quality-setting": "Qualité des textures",
```

Comment:
I'm not sure whether it is the textures or the lighting effects (Materials)

Reply:
This refers to materials/textures. e.g Low quality will not display metallic, roughness and normal maps.